compiler/rustc_abi/src/lib.rs

```diff
@@ -1468,7 +1461,6 @@ pub struct PointeeInfo {
     pub size: Size,
     pub align: Align,
     pub safe: Option<PointerKind>,
-    pub address_space: AddressSpace,
```

Comment:
Awesome, thanks a ton. :)
Note to self: add a comment to `PointeeInfo` stating that this is for optimization purposes only and safe to ignore by backends (as suggested by @eddyb)